### ISSUE TITLE
(BUG) [1.15.2]Crash upon putting sand if the Tropicraft sifter on server.

### ISSUE THREAD

**Issue by RVOtakuMike** [Tropicraft/Tropicraft#383]
## Issue Description

I am developing a private modpack and when a friend who was helping me test was testing Tropicraft, the sifter crashes the client, but only when connected to the server.
Server produces no errors.

## Steps to Reproduce
1) Go into creative on a server.
2) Get the sifter.
3) Put sand in sifter by right-clicking and wait about 3 seconds.

Tropicraft Version: Tropicraft-8.1.0-beta-34   
Forge Version: 1.15.2-forge-31.2.46   

## Crash log
```
---- Minecraft Crash Report ----
// Don't be sad, have a hug! <3

Time: 11/17/20 9:47 PM
Description: Ticking block entity

java.lang.NullPointerException: Ticking block entity
	at net.minecraftforge.fml.LogicalSidedProvider.lambda$static$3(LogicalSidedProvider.java:33) ~[?:?] {re:classloading}
	at net.minecraftforge.fml.LogicalSidedProvider.get(LogicalSidedProvider.java:60) ~[?:?] {re:classloading}
	at net.minecraftforge.fml.network.PacketDistributor.getServer(PacketDistributor.java:263) ~[?:?] {re:classloading}
	at net.minecraftforge.fml.network.PacketDistributor.lambda$playerListDimConsumer$2(PacketDistributor.java:219) ~[?:?] {re:classloading}
	at net.minecraftforge.fml.network.PacketDistributor$PacketTarget.send(PacketDistributor.java:178) ~[?:?] {re:classloading}
	at net.minecraftforge.fml.network.simple.SimpleChannel.send(SimpleChannel.java:124) ~[?:?] {re:classloading}
	at net.tropicraft.core.common.network.TropicraftPackets.sendToDimension(TropicraftPackets.java:41) ~[?:8.1.0-beta+34] {re:classloading}
	at net.tropicraft.core.common.block.tileentity.SifterTileEntity.syncInventory(SifterTileEntity.java:222) ~[?:8.1.0-beta+34] {re:classloading}
	at net.tropicraft.core.common.block.tileentity.SifterTileEntity.stopSifting(SifterTileEntity.java:176) ~[?:8.1.0-beta+34] {re:classloading}
	at net.tropicraft.core.common.block.tileentity.SifterTileEntity.func_73660_a(SifterTileEntity.java:70) ~[?:8.1.0-beta+34] {re:classloading}
	at net.minecraft.world.World.func_217391_K(World.java:473) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,xf:fml:observerlib:coremodmethod,xf:fml:astralsorcery:sun_brightness_server,pl:mixin:APP:assets/immersive_portals/immersive_portals.mixins.json:MixinWorld,pl:mixin:A}
	at net.minecraft.client.world.ClientWorld.func_217419_d(ClientWorld.java:164) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,xf:fml:xaeroworldmap:xaero_wm_clientworldclass,xf:fml:xaerominimap:xaero_clientworldclass,pl:mixin:APP:assets/immersive_portals/immersive_portals.mixins_client.json:alternate_dimension.MixinClientWorld_A,pl:mixin:APP:assets/immersive_portals/immersive_portals.mixins_client.json:MixinClientWorld,pl:mixin:APP:assets/immersive_portals/immersive_portals.mixins_ma_client.json:MixinClientWorld_MA,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1398) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,xf:fml:codechickenlib:IItemRenderer,pl:mixin:APP:assets/immersive_portals/immersive_portals.mixins_client.json:block_manipulation.MixinMinecraftClient_B,pl:mixin:APP:assets/immersive_portals/immersive_portals.mixins_client.json:MixinMinecraftClient,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_195542_b(Minecraft.java:904) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,xf:fml:codechickenlib:IItemRenderer,pl:mixin:APP:assets/immersive_portals/immersive_portals.mixins_client.json:block_manipulation.MixinMinecraftClient_B,pl:mixin:APP:assets/immersive_portals/immersive_portals.mixins_client.json:MixinMinecraftClient,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:553) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,xf:fml:codechickenlib:IItemRenderer,pl:mixin:APP:assets/immersive_portals/immersive_portals.mixins_client.json:block_manipulation.MixinMinecraftClient_B,pl:mixin:APP:assets/immersive_portals/immersive_portals.mixins_client.json:MixinMinecraftClient,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(SourceFile:204) ~[minecraft.jar:?] {re:classloading}
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_272] {}
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_272] {}
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_272] {}
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_272] {}
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:51) ~[forge-1.15.2-31.2.46-launcher.jar:31.2] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-5.1.2.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-5.1.2.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-5.1.2.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:81) [modlauncher-5.1.2.jar:?] {re:classloading}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:65) [modlauncher-5.1.2.jar:?] {re:classloading}
	at io.github.zekerzhayard.forgewrapper.installer.Main.main(Main.java:50) [ForgeWrapper-1.4.2.jar:?] {}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraftforge.fml.LogicalSidedProvider.lambda$static$3(LogicalSidedProvider.java:33)
	at net.minecraftforge.fml.LogicalSidedProvider.get(LogicalSidedProvider.java:60)
	at net.minecraftforge.fml.network.PacketDistributor.getServer(PacketDistributor.java:263)
	at net.minecraftforge.fml.network.PacketDistributor.lambda$playerListDimConsumer$2(PacketDistributor.java:219)
	at net.minecraftforge.fml.network.PacketDistributor$PacketTarget.send(PacketDistributor.java:178)
	at net.minecraftforge.fml.network.simple.SimpleChannel.send(SimpleChannel.java:124)
	at net.tropicraft.core.common.network.TropicraftPackets.sendToDimension(TropicraftPackets.java:41)
	at net.tropicraft.core.common.block.tileentity.SifterTileEntity.syncInventory(SifterTileEntity.java:222)
	at net.tropicraft.core.common.block.tileentity.SifterTileEntity.stopSifting(SifterTileEntity.java:176)
	at net.tropicraft.core.common.block.tileentity.SifterTileEntity.func_73660_a(SifterTileEntity.java:70)

-- Block entity being ticked --
Details:
	Name: tropicraft:sifter // net.tropicraft.core.common.block.tileentity.SifterTileEntity
	Block: Block{tropicraft:sifter}
	Block location: World: (588,70,-236), Chunk: (at 12,4,4 in 36,-15; contains blocks 576,0,-240 to 591,255,-225), Region: (1,-1; contains chunks 32,-32 to 63,-1, blocks 512,0,-512 to 1023,255,-1)
	Block: Block{tropicraft:sifter}
	Block location: World: (588,70,-236), Chunk: (at 12,4,4 in 36,-15; contains blocks 576,0,-240 to 591,255,-225), Region: (1,-1; contains chunks 32,-32 to 63,-1, blocks 512,0,-512 to 1023,255,-1)
Stacktrace:
	at net.minecraft.world.World.func_217391_K(World.java:473)
	at net.minecraft.client.world.ClientWorld.func_217419_d(ClientWorld.java:164)

-- Affected level --
Details:
	All players: 1 total; [ClientPlayerEntity['StarlightBeam'/83608, l='MpServer', x=587.90, y=70.00, z=-236.70]]
	Chunk stats: Client Chunks (ImmPtl) 529
	Level dimension: minecraft:overworld
	Level name: MpServer
	Level seed: 1544944681383654793
	Level generator: ID 00 - default, ver 1. Features enabled: false
	Level generator options: {}
	Level spawn location: World: (224,71,32), Chunk: (at 0,4,0 in 14,2; contains blocks 224,0,32 to 239,255,47), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 1545544 game time, 62513 day time
	Known server brands: 
	Level was modded: false
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
	Server brand: forge
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.client.world.ClientWorld.func_72914_a(ClientWorld.java:442)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:1763)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:567)
	at net.minecraft.client.main.Main.main(SourceFile:204)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:51)
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37)
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54)
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72)
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:81)
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:65)
	at io.github.zekerzhayard.forgewrapper.installer.Main.main(Main.java:50)

-- System Details --
Details:
	Minecraft Version: 1.15.2
	Minecraft Version ID: 1.15.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_272, AdoptOpenJDK
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), AdoptOpenJDK
	Memory: 4678297424 bytes (4461 MB) / 8589934592 bytes (8192 MB) up to 8589934592 bytes (8192 MB)
	CPUs: 12
	JVM Flags: 5 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms8192m -Xmx8192m -XX:+UseG1GC -XX:MaxGCPauseMillis=4
	ModLauncher: 5.1.2+70+master.2845bb9
	ModLauncher launch target: fmlclient
	ModLauncher naming: srg
	ModLauncher services: 
		/mixin-0.8.2.jar mixin PLUGINSERVICE 
		/eventbus-2.2.1-service.jar eventbus PLUGINSERVICE 
		/forge-1.15.2-31.2.46-launcher.jar object_holder_definalize PLUGINSERVICE 
		/forge-1.15.2-31.2.46-launcher.jar runtime_enum_extender PLUGINSERVICE 
		/forge-1.15.2-31.2.46-launcher.jar capability_inject_definalize PLUGINSERVICE 
		/accesstransformers-2.1.3-shadowed.jar accesstransformer PLUGINSERVICE 
		/forge-1.15.2-31.2.46-launcher.jar runtimedistcleaner PLUGINSERVICE 
		/mixin-0.8.2.jar mixin TRANSFORMATIONSERVICE 
		/forge-1.15.2-31.2.46-launcher.jar fml TRANSFORMATIONSERVICE 
	FML: 31.2
	Forge: net.minecraftforge:31.2.46
	FML Language Providers: 
		javafml@31.2
		minecraft@1
	Mod List: 
		CyclopsCore-1.15.2-1.8.3.jar Cyclops Core {cyclopscore@1.8.3 DONE}
		Tropicraft-8.1.0-beta-34.jar Tropicraft {tropicraft@8.1.0-beta+34 DONE}
		astralsorcery-1.15.2-1.12.11.jar Astral Sorcery {astralsorcery@1.15.2-1.12.11 DONE}
		NetherPortalFix-1.15.2-6.0.2.jar NetherPortalFix {netherportalfix@6.0.2 DONE}
		borderless-1.15.2-1.1.0.jar Borderless Window {borderless@1.15.2-1.1.0 DONE}
		ProjectE-1.15.2-PE1.0.4.jar ProjectE {projecte@1.0.4 DONE}
		mcwwin-1.15.2-v1.0.2.jar Macaw's Windows {mcwwindows@1.0.2 DONE}
		IndustrialAgriculture-[Core]-1.15.2-1.7.0.jar Industrial Agriculture {industrialagriculture@1.15.2-1.7.0 DONE}
		IronJetpacks-1.15.2-3.0.3.jar Iron Jetpacks {ironjetpacks@3.0.3 DONE}
		ForgeEndertech-1.15.2-6.0.4.0-build.0044.jar Forge Endertech {forgeendertech@6.0.4.0 DONE}
		XaerosWorldMap-1.11.0-Forge-1.15.2.jar Xaero's World Map {xaeroworldmap@1.11.0 DONE}
		CookingForBlockheads-1.15.2-8.0.3.jar Cooking for Blockheads {cookingforblockheads@8.0.3 DONE}
		Controlling-6.1.5.6.jar Controlling {controlling@6.1.5.6 DONE}
		ReAuth-1.14-1.15-3.8.1.jar ReAuth {reauth@3.8.1 DONE}
		Placebo-1.15.2-3.1.0.jar Placebo {placebo@3.1.0 DONE}
		citadel-1.1.11.jar Citadel {citadel@1.1.11 DONE}
		rangedpumps-0.7.1.jar Ranged Pumps {rangedpumps@0.7.1 DONE}
		mcwdoors-1.0.1fix-mc1.15.1-2.jar Macaw's Doors {mcwdoors@1.0.1 DONE}
		Morpheus-1.15.2-4.2.50.jar Morpheus {morpheus@version DONE}
		MekanismGenerators-1.15.2-9.10.9.422.jar Mekanism: Generators {mekanismgenerators@9.10.9 DONE}
		JustEnoughResources-1.15.2-0.10.1.77.jar Just Enough Resources {jeresources@0.10.1.77 DONE}
		XNetGases-1.15.2-1.0.0.jar XNet Gases {xnetgases@1.0.0 DONE}
		MysticalAgradditions-1.15.2-3.0.1.jar Mystical Agradditions {mysticalagradditions@3.0.1 DONE}
		refinedstorage-1.8.8.jar Refined Storage {refinedstorage@1.8.8 DONE}
		IndustrialAgriculture-[Botania]-1.15.2-1.7.0.jar Industrial Agriculture Plugin-Botania {iapb@1.15.2-1.7.0 DONE}
		PackMenu-1.15.2-1.2.8.jar Pack Menu {packmenu@1.2.8 DONE}
		BetterPingDisplay-1.15.2-1.0.jar Better Ping Display {betterpingdisplay@1.0 DONE}
		mcwbridges-1.0.4fix-mc1.15.2.jar Macaw's Bridges {mcwbridges@1.0.4 DONE}
		industrialforegoing-1.15.2-2.3.3-e356e61.jar Industrial Foregoing {industrialforegoing@2.3.3 DONE}
		BiomesOPlenty-1.15.2-10.0.0.366-universal.jar Biomes O' Plenty {biomesoplenty@1.15.2-10.0.0.366 DONE}
		phosphor-forge-mc1.15.2-0.5.2-build.4.jar Phosphor {phosphor@0.5.2+build.4 DONE}
		MekanismAdditions-1.15.2-9.10.9.422.jar Mekanism: Additions {mekanismadditions@9.10.9 DONE}
		mcwtrapdors-1.0.0fix-mc1.15.2.jar Macaw's Trapdoors {mcwtrpdoors@1.0.0 DONE}
		AudioСassettes-1.15.2-1.0.jar Audiocassettes {audiocassettes@1.0 DONE}
		portality-1.15.2-2.1.2-aaa257f.jar Portality {portality@2.1.2 DONE}
		curios-FORGE-1.15.2-2.0.2.6.jar Curios API {curios@FORGE-1.15.2-2.0.2.6 DONE}
		Botania-r1.15-388.jar Botania {botania@r1.15-388 DONE}
		Patchouli-1.15.2-1.2-35(1).jar Patchouli {patchouli@1.15.2-1.2-35 DONE}
		collective-1.15.2-1.53.jar Collective {collective@1.53 DONE}
		clothconfig-2-3.0.jar Cloth Config v2 API {cloth-config2@3.0 DONE}
		OreExcavation-1.7.151.jar Ore Excavation {oreexcavation@NONE DONE}
		youmatter-1.15.2-2.1.0.jar YouMatter {youmatter@1.15.2-2.1.0 DONE}
		obfuscate-0.5.0-1.15.2.jar Obfuscate {obfuscate@0.5.0 DONE}
		MekanismTools-1.15.2-9.10.9.422.jar Mekanism: Tools {mekanismtools@9.10.9 DONE}
		mcwroofs-1.15.2-v2.0.0.jar Macaw's Roofs {mcwroofs@2.0.0 DONE}
		furniture-7.0.0-pre16-1.15.1.jar MrCrayfish's Furniture Mod {cfm@7.0.0-pre16 DONE}
		cctweaked-1.15.2-1.91.0.jar CC: Tweaked {computercraft@1.91.0 DONE}
		observerlib-1.15.2-1.3.1.jar ObserverLib {observerlib@1.15.2-1.3.1 DONE}
		FastLeafDecay-v22.jar FastLeafDecay {fastleafdecay@v22 DONE}
		CodeChickenLib-1.15.2-3.3.2.383-universal.jar CodeChicken Lib {codechickenlib@3.3.2.383 DONE}
		IndustrialAgriculture-[Industrial-Foregoing]-1.15.2-1.6.1.jar Industrial Agriculture Plugin-Industrial Foregoing {iapif@1.15.2-1.6.1 DONE}
		doubledoors-1.15-2.1.jar Double Doors {doubledoors@2.1 DONE}
		Cucumber-1.15.2-3.0.9.jar Cucumber Library {cucumber@3.0.9 DONE}
		pamhc2trees-1.15.2-1.0.7.jar Pam's HarvestCraft 2 Fruit Trees {pamhc2trees@version DONE}
		flatbedrock-1.15.2-1.0.2.jar Flat Bedrock {flatbedrock@1.15.2-1.0.2 DONE}
		jei-1.15.2-6.0.3.15.jar Just Enough Items {jei@6.0.3.15 DONE}
		JustEnoughCalculation-1.15.2-3.6.2.jar Just Enough Calculation {jecalculation@3.6.2 DONE}
		Mekanism-1.15.2-9.10.9.422.jar Mekanism {mekanism@9.10.9 DONE}
		reap-1.12.4.jar Reap Mod {reap@1.12.4 DONE}
		invtweaks-1.15.2-2.0.9.jar Inventory Tweaks Renewed {invtweaks@1.15.2-2.0.9 DONE}
		Waystones-1.15.2-6.0.2.jar Waystones {waystones@6.0.2 DONE}
		WallpaperCraft-1.15.2-1.2.4.jar Wallpapercraft {wallpapercraft@1.2.4 DONE}
		WallpapercraftSlabs-1.15.2-1.1.0.jar Wallpapercraft - Slabs! {wallpapercraftslabs@1.1.0 DONE}
		WallpapercraftStairs-1.15.2-1.1.1.jar Wallpapercraft - Stairs {wallpapercraftstairs@1.1.1 DONE}
		Clumps-5.0.2.6.jar Clumps {clumps@5.0.2.6 DONE}
		lightoverlay-4.7.2.jar Light Overlay Forge {lightoverlay-forge@NONE DONE}
		FarmingForBlockheads-1.15.2-6.1.5.jar Farming for Blockheads {farmingforblockheads@6.1.5 DONE}
		pneumaticcraftrepressurized-1.15.2-1.4.5-69.jar PneumaticCraft: Repressurized {pneumaticcraft@1.15.2-1.4.5-69 DONE}
		mcjtylib-1.15-4.1.11.jar McJtyLib {mcjtylib@1.15-4.1.11 DONE}
		rftoolsbase-1.15-1.1.7.jar RFToolsBase {rftoolsbase@1.15-1.1.7 DONE}
		xnet-1.15-2.1.12.jar XNet {xnet@1.15-2.1.12 DONE}
		rftoolsstorage-1.15-1.1.11.jar RFToolsStorage {rftoolsstorage@1.15-1.1.11 DONE}
		rftoolscontrol-1.15-3.0.9.jar RFToolsControl {rftoolscontrol@1.15-3.0.9 DONE}
		mahoutsukai-1.15.2-v1.25.2.jar Mahou Tsukai {mahoutsukai@1.15.2-v1.25.2 DONE}
		iceandfire-2.1.0-1.15.2.jar Ice and Fire {iceandfire@2.0.3-1.15.2 DONE}
		ToastControl-1.15.2-3.0.1.jar Toast Control {toastcontrol@3.0.1 DONE}
		MCDJ-1.15.2-2.1.jar MCDJ {mcdj@2.1 DONE}
		immersiveportals-0.24forMc1.15withForge.jar Immersive Portals {immersive_portals@0.24 DONE}
		CraftTweaker-1.15.2-6.0.0.24.jar CraftTweaker {crafttweaker@6.0.0.24 DONE}
		rftoolspower-1.15-2.2.6.jar RFToolsPower {rftoolspower@1.15-2.2.6 DONE}
		extradisks-1.15.2-1.2.2.jar Extra Disks {extradisks@1.15.2-1.2.2 DONE}
		forge-1.15.2-31.2.46-universal.jar Forge {forge@31.2.46 DONE}
		PassthroughSigns-1.15.2-2.3.5.jar Passthrough Signs {passthroughsigns@1.15.2-2.3.5 DONE}
		immersiveposts-1.15.2-2.2.0.jar Immersive Posts {immersiveposts@2.2.0 DONE}
		MysticalAgriculture-1.15.2-3.0.10.jar Mystical Agriculture {mysticalagriculture@3.0.10 DONE}
		ironchest-1.15.2-10.0.3.jar Iron Chests {ironchest@1.15.2-10.0.1 DONE}
		forge-1.15.2-31.2.46-client.jar Minecraft {minecraft@1.15.2 DONE}
		cofhcore-1.15.2-1.0.2.jar CoFH Core {cofh_core@1.0.2 DONE}
		thermal-1.15.2-1.0.2.jar Thermal Series {thermal@1.0.2 DONE}
		thermalinnovation-1.15.2-1.0.0.jar Thermal Innovation {thermal_innovation@1.0.0 DONE}
		thermalexpansion-1.15.2-1.0.2.jar Thermal Expansion {thermal_expansion@1.0.2 DONE}
		thermallocomotion-1.15.2-1.0.0.jar Thermal Locomotion {thermal_locomotion@1.0.0 DONE}
		rftoolsutility-1.15-2.1.15.jar RFToolsUtility {rftoolsutility@1.15-2.1.15 DONE}
		Flopper-1.15.2-1.0.6.jar Flopper {flopper@1.0.6 DONE}
		MouseTweaks-2.13-mc1.15.1.jar Mouse Tweaks {mousetweaks@2.13 DONE}
		titanium-1.15.2-2.4.2.jar Titanium {titanium@2.4.2 DONE}
		ImmersiveEngineering-1.15.2-4.1.1-125.jar Immersive Engineering {immersiveengineering@1.15.2-4.1.1-125 DONE}
		pamhc2crops-1.15.2-1.0.5.jar Pam's HarvestCraft 2 Crops {pamhc2crops@version DONE}
		Ding-1.15.2-1.1.0.jar Ding {ding@1.1.0 DONE}
		ChickenChunks-1.15.2-2.5.1.79-universal.jar ChickenChunks {chickenchunks@1.15.2-2.5.1.79 DONE}
		rftoolsbuilder-1.15-2.1.12.jar RFToolsBuilder {rftoolsbuilder@1.15-2.1.12 DONE}
		jeiintegration-1.15.2-3.1.1.2.jar JEI Integration {jeiintegration@3.1.1.2 DONE}
		XaerosMinimap-20.27.2-Forge-1.15.2.jar Xaero's Minimap {xaerominimap@20.27.2 DONE}
		JAOPCA-1.15.2-3.2.0.15.jar JAOPCA {jaopca@3.2.0.15 DONE}
		pamhc2foodcore-1.15.2-1.1.0.jar Pam's HarvestCraft 2 Food Core {pamhc2foodcore@version DONE}
		thermalcultivation-1.15.2-1.0.2.jar Thermal Cultivation {thermal_cultivation@1.0.2 DONE}
		serverconfigupdater-1.1.jar ServerConfig Updater {serverconfigupdater@1.1 DONE}
		FastWorkbench-1.15.2-3.1.1.jar FastWorkbench {fastbench@3.1.1 DONE}
		SecurityCraft-v1.8.20.2.jar SecurityCraft {securitycraft@v1.8.20.2 DONE}
		AutoRegLib-1.5-40.jar AutoRegLib {autoreglib@1.5-40 DONE}
		Quark-r2.1-245.jar Quark {quark@r2.1-245 DONE}
		FluxNetworks-1.15.2-5.0.3-4.jar Flux Networks {fluxnetworks@1.15.2-5.0.3-4 DONE}
		ironfences-1.15.2-1.0.0.jar Iron fences {ironfencegates@1.0.0 DONE}
		FastFurnace-1.15.1-3.0.0.jar FastFurnace {fastfurnace@3.0.0 DONE}
		IndustrialAgriculture-[Mekanism]-1.15.2-1.6.1.jar Industrial Agriculture Plugin-Mekanism {iapmekanism@1.15.2-1.6.1 DONE}
		refinedstorageaddons-0.6.3.jar Refined Storage Addons {refinedstorageaddons@0.6.3 DONE}
		thutcore-1.15.2-7.2.1.jar Thut Core {thutcore@7.2.1 DONE, thutcrafts@7.2.1 DONE}
		thuttech-1.15.2-8.2.2.jar Thut Elevators {thuttech@8.2.2 DONE}
		MorphoTool-1.3-23.jar Morph-o-Tool {morphtool@1.3-23 DONE}
	Patchouli open book context: n/a
	Launched Version: 1.15.2-forge-31.2.46
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: Intel(R) UHD Graphics GL version 4.6.0 - Build 26.20.100.7642, Intel
	GL Caps: Using framebuffer using OpenGL 3.0
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'forge'
	Type: Client (map_client.txt)
	Resource Packs: 
	Current Language: English (US)
	CPU: 12x Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz
```

**Comment by NotActuallyTerry** [comment]
Confirmed in 1.15, was fixed in 1.16 by ae4e4d1

**Comment by NotActuallyTerry** [comment]
Fix has been pushed to 1.15
